juriscraper/opinions/united_states/state/pasuperct.py

```diff
@@ -7,11 +7,33 @@
 Date created: 21 July 2014
 """
 
+from datetime import datetime
+from typing import Dict
+from urllib.parse import urlencode
+
 from juriscraper.opinions.united_states.state import pa
 
 
 class Site(pa.Site):
+    court = "Superior"
+    days_interval = 20
+    first_opinion_date = datetime(1998, 2, 15)
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
-        self.url = "https://www.pacourts.us/Rss/Opinions/Superior/"
+        self.params["postTypes"] = "1,2,3,4,5,7,8,9,13,14,15,16,17,18"
```

Comment:
I think this should be 

1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,32,33

here they list all memorandum opinions as non-precedential - and dont use the opinion but I think that is what they are -  not Sure what complete and revised definitively mean